.github/ISSUE_TEMPLATE/tracking_issue.md

```diff
@@ -0,0 +1,41 @@
+---
+name: 🔍 Tracking Issue
+about: Create an issue that tracks a wider effort
+title: 'Tracking Issue for ...'
+labels: ["C-tracking-issue"]
+assignees: ''
+
+---
+
+<!-- Consider keeping the following section in the issue. -->
+### About tracking issues
+Tracking issues are used to record the overall progress of implementation.
+They are also used as hubs connecting to other relevant issues, e.g., bugs or open design questions.
+A tracking issue is however not meant for large scale discussion, questions, or bug reports about a feature.
+Instead, open a dedicated issue for the specific matter.
+
+### What
```

Comment:
These are `###` instead of `##` like other issue templates because:
1. `rust-lang/rust` does it like this
2. It makes this template stand-out